### DISCUSSION TITLE
README of zh-TW is being translated into zh-CN

### DISCUSSION THREAD

**Discussion by CSY54** [supabase/supabase#16844] [Contribute to Supabase]
# Improve documentation

## Link 

https://github.com/supabase/supabase/blob/master/i18n/README.zh-tw.md

## Describe the problem

As title, the zh-TW translation is being translated into zh-CN.

It seems that #14359 has modified the translation files using DeepL, a machine learning translation service. However, it only supports Simplified Chinese, but not Traditional Chinese. Also, from the repo the issue mentioned, the translated file only contains [README.zh.md](https://github.com/lyqht/deepL-demo/blob/master/docs/supabase/README.zh.md), which is Simplified Chinese.

## Describe the improvement

Manually translate the zh-TW one, as DeepL seems to fail the job.

## Additional context

DeepL only supports Simplified Chinese (简体中文), but not Traditional Chinese (繁體中文).
<img width="336" alt="DeepL" src="https://github.com/supabase/supabase/assets/18496305/602ee42f-7055-4d4b-8c0b-91a2aae7c594">

**Comment by KSSaiTeja** [comment]
can i work on this issue

**Comment by gregnr** [comment]
That would be great @KSSaiTeja!

**Comment by KSSaiTeja** [comment]
that's great! could you please tell me what to do more clearly and which docs to follow and also provide links for docs. it would be more easy for me
